process/README.md

```diff
@@ -3,4 +3,5 @@
 Guides covering how we do things
 
 - [Pull requests](pull_request.md)
+- [Releases](releases.md)
```

Comment:
You also need to make this same change on the root README document.

Reply:
🤦‍♂️ 👍